crates/biome_css_analyze/src/keywords.rs

```diff
@@ -81,3 +81,679 @@ pub const FONT_SIZE_KEYWORDS: [&str; 9] = [
 ];
 
 pub const LINE_HEIGHT_KEYWORDS: [&str; 1] = ["normal"];
+
+/// List of known CSS value functions sourced from [`css-functions-list`](https://www.npmjs.com/package/css-functions-list).
+/// See the original list [here](https://github.com/niksy/css-functions-list/blob/master/index.json).
+pub const FUNCTION_KEYWORDS: [&str; 671] = [
```

Comment:
These are ordered entries. We should use `binary_search` instead of `contains` (it's faster).

Although, entries must be ordered. So we should add some tests for that. We do the same with other constants

Reply:
I have sorted the entries and updated the code to use `binary_search` instead of `contains`.